control-plane/agents/core/src/node/specs.rs

```diff
@@ -141,4 +141,29 @@ impl ResourceSpecsLocked {
             })
             .collect()
     }
+
+    /// Drain the node with the given ID.
```

Comment:
We're not actually draining the node here?

Reply:
I've amended the comments.